app/controllers/application_controller.rb

```diff
@@ -14,19 +14,11 @@ class ApplicationController < ActionController::Base
 
   helper_method :blackcat_config
 
-  before_action :flash_readonly, :flash_alert
-
-  def blackcat_config
-    @blackcat_config ||= BlackcatConfig::Builder.new
-  end
+  before_action :flash_alert
 
   private
 
-    def flash_readonly
-      flash.now[:error] = blackcat_config.blackcat_message(:readonly) if blackcat_config.blackcat_message?(:readonly)
-    end
-
     def flash_alert
-      flash.now[:error] = blackcat_config.blackcat_message(:alert) if blackcat_config.blackcat_message?(:alert)
+      flash.now[:error] = ENV['alert']
```

Comment:
same here, needs to be sanitized

maybe this?

```suggestion
      flash.now[:error] = Rails::Html::LinkSanitizer.new.sanitize(ENV['alert'])
```

Reply:
got rid of flash alert entirely 👌